src/languageProvider/locales/en_US.json

```diff
@@ -57,12 +58,20 @@
   "create.validResultSetStart": "Must be greater than or equal to Betting End Time",
   "depositDialog.title": "QTUM/BOT Deposit Address",
   "eventHistory.viewEvent": "View Event",
+  "eventInfo.creator": "CREATOR",
+  "eventInfo.endDate": "ENDING DATE",
+  "eventInfo.fund": "FUNDING",
+  "eventInfo.predictionFund": "PREDICTION FUNDING",
+  "eventInfo.resultSetter": "RESULT SETTER",
+  "eventInfo.voteVolumn": "VOTING VOLUME",
```

Comment:
lets change all of these CAPS strings to regular casing:
creator: "Creator",
endingDate: "Ending Date",
etc.

Then in the actual code where this is being used, we can use `endingDate.toUpperCase()`.

This will allow us to reuse strings whenever possible and not worry about the capitalization. Otherwise we would have to have 2 diff strings for like `BET` and `Bet` which makes no sense since they are the same word.